packages/sdk/src/syncedStreams.ts

```diff
@@ -589,6 +593,30 @@ export class SyncedStreams {
         }
     }
 
+    private syncDown(streamId: Uint8Array): void {
+        if (streamId === undefined || streamId.length === 0) {
+            this.logError('syncDown: streamId is empty')
+            return
+        }
+        if (this.syncState !== SyncState.Syncing) {
+            this.logError('syncDown: invalid state transition', this.syncState)
+            return
+        }
+        const stream = this.streams.get(streamIdAsString(streamId))
+        if (!stream) {
+            this.log('syncDown: stream not found', streamIdAsString(streamId))
+            return
+        }
+        const cookie = stream.view.syncCookie
+        if (!cookie) {
+            this.logError('syncDown: syncCookie not found', streamIdAsString(streamId))
+            return
+        }
+        this.addStreamToSync(cookie).catch((err) => {
```

Comment:
If a stream is down it is likely down for some time. For instance when a node restarts after a release upgrade. Adding it immediately after it's reported down will likely fail. 

Not sure if this retry mechanism is something that the sdk needs to support or if it is something that the app using the sdk must do. I do think that the sdk should have the ability to report that it a stream is down. This allows an app to report to the user that some streams won't get updates.